### PR TITLE
Enable rating-server to access config server via eureka discovery.

### PR DESCRIPTION
**Pull request by alex-muliarevych**
Enable connect of rating server to config server not via default settings but using eureka discovery.
Wasn't able to test with dockers test branch.

### CODE REVIEW

File: rating-service/src/main/resources/bootstrap.properties [hunk 1]
```diff
@@ -0,0 +1,3 @@
+spring.cloud.config.discovery.enabled=true
```
Comment (banadiga) [inline rating-service/src/main/resources/bootstrap.properties:1]: Can we move all properties in application.properties ?

File: rating-service/src/main/resources/bootstrap.properties [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+spring.cloud.config.fail-fast=true
+spring.cloud.config.discovery.serviceId=config-server
```
Comment (banadiga) [inline rating-service/src/main/resources/bootstrap.properties]: I prefer to add '\n' into the end of file

**Review by banadiga — APPROVED** [review]
Can you add the same properties to the `hackster-service`?

Good job. Thanks for the help.

File: rating-service/src/main/resources/bootstrap.properties (hunk 1 above)
Comment (alex-muliarevych) [reply rating-service/src/main/resources/bootstrap.properties:1]: Unfortunately not, bootstrap file is loaded as first one, if we put properties to application.
FIrstly will be loaded anyway default link with port 8888, checked such effect when set configuration server to some custom port as 9999 i.e.. In such case works only if properties of configuration server are read from bootstrap property file.

File: rating-service/src/main/resources/bootstrap.properties (hunk 2 above)
Comment (alex-muliarevych) [reply rating-service/src/main/resources/bootstrap.properties]: Added additional newline, but on github it isn't visible in change view.

**Comment by alex-muliarevych** [comment]
Added configuration for hackster-service also.

File: rating-service/src/main/resources/bootstrap.properties (hunk 1 above)
Comment (banadiga) [reply rating-service/src/main/resources/bootstrap.properties:1]: :+1:

**Pull request merged**